pkgs/development/libraries/libfabric/default.nix

```
@@ -1,24 +1,36 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, autoreconfHook, libpsm2
-, enablePsm2 ? (stdenv.isx86_64 && stdenv.isLinux) }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, autoreconfHook
+, libpsm2
+, libuuid
+, numactl
+, enablePsm2 ? (stdenv.isx86_64 && stdenv.isLinux)
+, enableOpx ? (stdenv.isx86_64 && stdenv.isLinux)
+}:
```

New convention is to group inputs with their switches:
```suggestion
, enablePsm2 ? (stdenv.isx86_64 && stdenv.isLinux)
, libpsm2
, enableOpx ? (stdenv.isx86_64 && stdenv.isLinux)
, libuuid
, numactl
}:
```